iron-core/src/main/java/io/axway/iron/core/spi/file/FileSnapshotStoreFactoryBuilder.java

```diff
@@ -0,0 +1,26 @@
+package io.axway.iron.core.spi.file;
+
+import java.nio.file.Path;
+import java.util.function.*;
+import javax.annotation.*;
+import io.axway.iron.spi.storage.SnapshotStoreFactory;
+
+public class FileSnapshotStoreFactoryBuilder implements Supplier<SnapshotStoreFactory> {
+    private Path m_dir;
+    private Integer m_limitedSize = 20;
```

Comment:
better name for that ? eg transactionIdPaddingLenght or a better proposition ?

Reply:
transactionIdPaddingLength it is now